cfme/cloud/instance/__init__.py

```diff
@@ -176,318 +347,207 @@ def find_quadicon(self, *args, **kwargs):
+                                   self.obj.provider.name))
+            row = None
+        if not row:
+            logger.warning('Attempting to match instance using name only: "{}"'
+                           .format(self.obj.name))
+            try:
+                row = self.prerequisite_view.paginator.find_row_on_pages(
+                    self.prerequisite_view.entities.table, name=self.obj.name)
+                self.log_message('Matched table row on instance name only')
+            except NoSuchElementException:
+                raise InstanceNotFound('Could not match instance by name only: {}'
+                                       .format(self.obj.name))
+
+        if row:
+            row.click()
+
+    def resetter(self, *args, **kwargs):
+        self.view.toolbar.reload.click()
 
 
 @navigator.register(Instance, 'Edit')
-class InstanceEdit(CFMENavigateStep):
+class Edit(CFMENavigateStep):
     prerequisite = NavigateToSibling('Details')
 
     # No am_i_here because the page only indicates name and not provider
 
-    def step(self):
-        cfg_btn('Edit this instance')
-
-
-@navigator.register(Instance, 'SetOwnership')
-class InstanceOwnership(CFMENavigateStep):
-    prerequisite = NavigateToSibling('Details')
-
-    # No am_i_here because the page only indicates name and not provider
-
-    def step(self):
-        cfg_btn('Set Ownership')
+    def step(self, *args, **kwargs):
+        self.prerequisite_view.toolbar.configuration.item_select('Edit this Instance')
 
 
 @navigator.register(Instance, 'EditMgmtEngineRelation')
-class InstanceEditMgmtEngineRelation(CFMENavigateStep):
-    prerequisite = NavigateToSibling('Details')
-
-    # No am_i_here because the page only indicates name and not provider
-
-    def step(self):
-        cfg_btn('Edit Management Engine Relationship')
-
-
-@navigator.register(Instance, 'AttachCloudVolume')
-class InstanceAttachVolume(CFMENavigateStep):
+class EditMgmtEngineRelation(CFMENavigateStep):
     prerequisite = NavigateToSibling('Details')
 
     # No am_i_here because the page only indicates name and not provider
-
-    def step(self):
-        cfg_btn('Attach a Cloud Volume to this instance')
+    def step(self, *args, **kwargs):
+        self.prerequisite_view.toolbar.configuration.item_select(
+            'Edit Management Engine Relationship')
 
 
-@navigator.register(Instance, 'DetachCloudVolume')
-class InstanceDetachVolume(CFMENavigateStep):
-    prerequisite = NavigateToSibling('Details')
+@navigator.register(Instance, 'Provision')
+class Provision(CFMENavigateStep):
+    VIEW = ProvisionView
+    prerequisite = NavigateToSibling('All')
 
-    # No am_i_here because the page only indicates name and not provider
+    def step(self, *args, **kwargs):
+        self.prerequisite_view.toolbar.lifecycle.item_select('Provision Instances')
 
-    def step(self):
-        cfg_btn('Detach a Cloud Volume to this instance')
+    def am_i_here(self, *args, **kwargs):
+        return False
```

Comment:
Why did you implement `am_i_here()` if there is `is_displayed` in `ProvisionView`?